llm/dyn_ext_server.c

```diff
@@ -11,16 +12,24 @@
 #define UNLOAD_LIBRARY(handle) dlclose(handle)
 #elif _WIN32
 #include <windows.h>
-#define LOAD_LIBRARY(lib, flags) LoadLibrary(lib)
+#define LOAD_LIBRARY(lib, flags) ({ \
```

Comment:
it's convention to define c macros with `do { ... } while (0)`, e.g.

```c
#define LOAD_LIBRARY(lib, flags) do { \
...                                   \
} while (0)
```

Reply:
These needs a return value, so I broke them out into separate functions instead. Let me know how that looks.